cli/switch.go

```diff
@@ -0,0 +1,35 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/bitrise-plugins-analytics/configs"
+	log "github.com/bitrise-io/go-utils/log"
+	"github.com/urfave/cli"
+)
+
+type onOff bool
+
+func (o onOff) String() string {
+	if o == true {
+		return "on"
+	}
+	return "off"
+}
+
+func createSwitchCommand(on onOff) cli.Command {
```

Comment:
a switch has "phases" or "states". please rename the type onOff to state and then you don't need to use `on onOff`. you can use for example `s state` and instead of `on.String` (that can be `off`) you can use `state.Stirng()`